pyffi/spells/nif/optimize.py

```diff
@@ -508,8 +508,7 @@ def branchentry(self, branch):
         # recalculate tangent space (only if the branch already exists)
         if (branch.find(block_name=b'Tangent space (binormal & tangent vectors)',
                         block_type=NifFormat.NiBinaryExtraData)
-            or (data.num_uv_sets & 61440)
-            or (data.bs_num_uv_sets & 61440)):
```

Comment:
This has been refactored to "Extra Vectors Flags" - https://github.com/niftools/nifxml/commit/3dd61a8b2c544de250f1d8693ba63158a0d873f9

It shold be data.extra_vectors_flags